test/functional/editor/K_spec.lua

```diff
@@ -33,6 +34,11 @@ describe('K', function()
     feed('i'..test_file..'<ESC>K')
     retry(nil, nil, function() eq(1, eval('filereadable("'..test_file..'")')) end)
     eq({'fnord'}, eval("readfile('"..test_file.."')"))
+    -- Confirm that Neovim is still in terminal mode after K is pressed (#16692).
+    sleep(5000) -- TODO: lower this if it works
```

Comment:
I'd recommend 500 here. Meanwhile there are some problems with GitHub's servers.
```suggestion
    sleep(500)
```

Reply:
Done (a81b5837027c98e199374bfaadbf5812601c7a44). Thanks!